### DISCUSSION TITLE
how to add icon in dialogue

### DISCUSSION THREAD

**Discussion by samaremad** [serenity-is/Serenity#5566] [Help]
hi everyone 
i want to add icon in dialouge(form) beside the "save" and "delete" icons in the master details page how can i do that?

**Comment by muhammedyaman** [comment]
`$("<i class='fa fa-check-circle'></i>").insertAfter(".delete-button");`
use this in xyzDialog.ts and because it is after delete-button (delete-button is not visible in new) you might want to check wether its new or not with `this.isNew()`

**Reply by samaremad** [reply]
i didnt get you well can you explain it again

**Reply by muhammedyaman** [reply]
The delete button only appears if the entity is not new. So if you want to append an icon after delete button you might have to check if the entity is new. 
```
protected afterLoadEntity(){
Super.afterLoadEntity();
if(!this.isNew()){
$("<i class='fa fa-check-circle'></i>").insertAfter(".delete-button");
}
```
